server/src/main/java/org/apache/druid/server/security/AuthorizerNameValidator.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server.security;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import org.apache.druid.java.util.common.StringUtils;
+
+import java.util.function.Consumer;
+
+/**
+ * A class that validates the name of an authorizer.
+ */
+public class AuthorizerNameValidator implements Consumer<String>
+{
+  private static final String AUTHORIZER_NAME = "authorizerName";
```

Comment:
It would be good to also use this constant in `AuthorizerResourceFilter.filter` and in the `@PathParam` annotations of BasicAuthorizerResource. There's a lot of places where the string "authorizerName" appears, and using a constant would emphasize that this specific string is important.